src/authorization.rs

```diff
@@ -214,8 +214,8 @@ impl Authorization {
 }
 
 impl Challenge {
-  /// The key authorization is the token that the HTTP01, or DNS01
-  // challenges should be serving for the ACME server to inspect.
+  /// The key authorization is the token that the HTTP01 challenge
+  // should be serving for the ACME server to inspect.
```

Comment:
```suggestion
  /// should be serving for the ACME server to inspect.
```

Drive by fix (same below)